### ISSUE TITLE
tfrecord convert to image for data recheck

### ISSUE THREAD

**Issue by Alexander-flyer** [A-bone1/FSNS-tfrecord-generate#2]
I have something wrong with attention-ocr training, so I wanna to make sure there is no mistake with my tfrecord file.
And the following code is my tf-reader, the tfrecord-generate is the same as you provide. could you take a look at it for me, or show me your tf-reader?
     
    def tf_to_img(data_files, num):
    try:
        # Read TFRecord file
        reader = tf.TFRecordReader()
        filename_queue = tf.train.string_input_producer(data_files)
        _, serialized_example = reader.read(filename_queue)

        # Define features
        read_features = {
            'image/encoded': tf.FixedLenFeature([], dtype=tf.string),
            'image/format': tf.FixedLenFeature([], dtype=tf.string),
            'image/width': tf.FixedLenFeature([], dtype=tf.int64),
            'image/orig_width': tf.FixedLenFeature([], dtype=tf.int64),
            'image/class': tf.FixedLenFeature([], dtype=tf.int64),
            'image/unpadded_class': tf.FixedLenFeature([], dtype=tf.int64),
            'image/text': tf.FixedLenFeature([], dtype=tf.string),
        }

        # Extract features from serialized data
        read_data = tf.parse_single_example(serialized=serialized_example,
                                            features=read_features)

        label = tf.cast(read_data['image/text'], tf.string)
        image = tf.decode_raw(read_data['image/encoded'], tf.uint8)
        image = tf.reshape(image, [150, 600, 3])
        with tf.Session() as sess:
            init_op = tf.initialize_all_variables()
            sess.run(init_op)
            coord = tf.train.Coordinator()
            threads = tf.train.start_queue_runners(coord=coord)
            for i in range(num):
                image, label = sess.run([image, label])
                img = Image.fromarray(image, 'RGB')
                img.show()
                print(label)
            coord.request_stop()
            coord.join(threads)
    except Exception as e:
        print(e)
    
    if __name__ == '__main__':
    data_path = '/Users/saber/PycharmProjects/attention_ocr/python/datasets/data/bankcard/test-bankcard-00*'
    data_files = tf.gfile.Glob(data_path)
    print(data_files)
    tf_to_img(data_files, 1145)

the error infor:

    InvalidArgumentError (see above for traceback): Key: image/class.  Can't parse serialized Example.
	 [[node ParseSingleExample/ParseSingleExample (defined at /Users/saber/PycharmProjects/attention_ocr/python/data_provider.py:258)  = ParseSingleExample[Tdense=[DT_INT64, DT_STRING, DT_STRING, DT_INT64, DT_STRING, DT_INT64, DT_INT64], dense_keys=["image/class", "image/encoded", "image/format", "image/orig_width", "image/text", "image/unpadded_class", "image/width"], dense_shapes=[[], [], [], [], [], [], []], num_sparse=0, sparse_keys=[], sparse_types=[], _device="/job:localhost/replica:0/task:0/device:CPU:0"](ReaderReadV2:1, ParseSingleExample/Const, ParseSingleExample/Const_1, ParseSingleExample/Const_1, ParseSingleExample/Const, ParseSingleExample/Const_1, ParseSingleExample/Const, ParseSingleExample/Const)]]

**Comment by Alexander-flyer** [comment]
a. problem have been resolved by only read `encoded` and  `text` features
